build/yamls/windows/containerd-with-ovs/kustomization.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../containerd
+patchesStrategicMerge:
+- containerd-with-ovs.yml
+commonLabels:
+  app: antrea
+configMapGenerator:
+- files:
+  - conf/Run-AntreaOVS-Containerd.ps1
```

Comment:
Should we add "Install-WindowsCNI-Containerd.ps1" ?

Reply:
It's already added from containerd resource. Only additional changes are present here to merge in the yaml generated from containerd.